gazupublisher/utils/software.py

```diff
@@ -42,6 +62,16 @@ def list_cameras():
             ("CameraStandAlone2", "CameraObject2"),
         ]
 
+
+def get_current_color_space():
+    import gazupublisher.working_context as w
+
+    if w.working_context == "BLENDER":
```

Comment:
if you are using an home made module, you could use a function like `is_blender_context`.

Reply:
Ok thx for the feedback, I'll make sure to update it in the next PR's